scripts/NCBI_Assembly/format_NCBI_Chromosome.py

```diff
@@ -0,0 +1,206 @@
+Name: format_NCBI_Chromosome
+Description:  Converts an NIH NCBI assembly reports file on a genome assembly 
+	into a mcf output file populating information on GenomeAssembly, 
+	GenomeAssemblyUnit, and Chromosome.
+
+@file_input	path to the input NCBI file on a genome assembly
+@file_output	path to the output mcf file to write the reformatted data
+@genome 	the shorthand name for the genome assembly represented in the 
+		file (e.g. hg38, mm10)
+@species_abrv	the abbreviation used by Data Commons to represent species dcid 
+		(e.g. hs, mm)
+'''
+
+import re
+
+def format_camel_case(value):
+	'''
+	Format string to camel case.
+
+	@value	The string that needs to be formatted to camel case
+	@return	The camel case formatted string
+	'''
+	value = value.replace(' ', '')
+	value_split = re.split('_|-', value)
+	value_formatted = ''
+	# camel case enum type
+	for i in range(len(value_split)):
+		value_formatted = value_formatted + value_split[i][0].upper() + \
+		value_split[i][1:]
+	return(value_formatted)
+
+
+def write_genome_assembly(file_output, dict_genome_assembly, genome, 
+	species_abrv):
+	'''
+	Write the information on the genome assembly to a GenomeAssembly node in the
+	mcf output file.
+
+	@file_output		path to the output mcf file to write the reformatted
+				data
+	@dict_genome_assembly	dictionary of the data fields and values containing 
+				information on the genome assembly
+	@genome 		the shorthand name for the genome assembly 
+				represented in the file (e.g. hg38, mm10)
+	@species_abrv		the abbreviation used by Data Commons to represent 
+				species dcid (e.g. hs, mm)
+	'''
+	dict_conversion = {'Assembly name': \
+	'genomeReferenceConsortiumAssemblyName',\
+	'Description': 'description',\
+	'Taxid': 'ncbiTaxonID',\
+	'BioProject': 'ncbiBioProject',\
+	'Submitter': 'submitter',\
+	'Date': 'date',\
+	'Assemblytype': 'genomeAssemblyType',\
+	'Releasetype': 'genomeAssemblyReleaseType',\
+	'Assemblylevel': 'genomeAssemblyLevel',\
+	'Genomerepresentation': 'isGenomeRepresentationFull',\
+	'RefSeqcategory': 'refSeqCategory',\
+	'GenBankassemblyaccession': 'genBankAssemblyAccession',\
+	'RefSeqassemblyaccession': 'refSeqAssemblyAccession',\
+	'RefSeqassemblyandGenBankassembliesidentical': \
+	'isRefSeqGenBankAssembliesIdentical',\
+	'Assemblyname': 'ncbiAssemblyName',\
+	'Infraspecificname': 'infraspecificName',\
+	'Synonyms': 'alternativeName',\
+	'WGSproject': 'wgsProject',\
+	'Assemblymethod:': 'assemblyMethod',\
+	'GenomeCoverage': 'genomeCoverage',\
+	'Sequencingtechnology': 'sequencingTechnology',
+	'Organismname': 'ofSpecies'}  # convert NCBI fields to property names
+	list_key_enums = ['Assemblytype', 'Releasetype', 'RefSeqcategory', \
+	'Assemblylevel']
+	w = open(file_output, mode='a')
+	w.write('Node: dcid:bio/' + genome + '\n')
+	w.write('name: "' + genome + '"\n')
+	w.write('typeOf: dcs:GenomeAssembly\n')
+	w.write('ofSpecies: dcid:bio/' + species_abrv + '\n')
+	for key, value in dict_genome_assembly.items():
```

Comment:
```suggestion
	for key, value in sorted(dict_genome_assembly.items()):
```

Its good practice to sort the keys so the output MCF is deterministic (e.g., when you compare against expected output it will be meaningful)